apps/sms/services/js/messaging/messaging_client.js

```diff
@@ -0,0 +1,110 @@
+/* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+
+
+/* global bridge, EventDispatcher */
+
+/* export MessagingClient */
+
+'use strict';
+(function(exports) {
+
+const EVENTS = [
+  'message-sending', 'message-failed-to-send', 'message-delivered',
+  'message-read', 'message-sent', 'message-received', 'threads-deleted'
+];
+
+var client = bridge.client('messaging');
+
+var MessagingClient = {
+  relayEvents: function msc_relayEvents() {
```

Comment:
nit: we don't need to name the functions anymore; and you can use the newer simpler structure `relayEvents() {}`